src/components/Controllers/View/store.ts

```diff
@@ -1,8 +1,9 @@
 import * as React from 'react'
 import * as zustand from 'zustand'
 import create from 'zustand/vanilla'
+import { Parser } from 'node-sql-parser'
```

Comment:
I'm using this library because of the Typescript support and it's easier to check for errors and referenced tables and fields after your parse a query